shared/minime/contracts/MinimeToken.sol

```diff
@@ -0,0 +1,575 @@
+pragma solidity ^0.4.24;
```

Comment:
Curious... [there's a minime v2 from last year](https://github.com/Giveth/minime/blob/master/contracts/MiniMeToken.sol); should we update to that?